### ISSUE TITLE
More Questions

### ISSUE THREAD

**Issue by eshau** [aws-neuron/nki-moe#17]
Hi!

1. Can we assume max model length is 1024 during eval?

Edit:
Some more questions:
2. Will the baseline model / logits be run on CPU or Trainium? Can we assume divergence-difference-tolerance will be 0.001?
3. Further clarification on quantization:

Let's say we were to use a quantized FP8 model with medusa decoding.
- I assume our correctness would be compared against a quantized FP8 model with medusa decoding.
- Would quantizing via `nki.isa.quantize_mx` count as part of tracked time or can we do that before and load those quantized weights?
- Would our latency and throughput be compared to the fixed argument numbers or that of the quantized FP8 model with medusa decoding?

4. Are we allowed to use `neuronx-distributed` / `nki-library` / `nki-samples` kernels? Would they be considered for NKI MACs?

5. Can we use NKIPy?

6. Where did `nki.simulate_kernel` go? Is it coming back? If we use the old `simulate_kernel`, will it still work?

7. Is there a way to speed up compilation?

8. Is there a way to compile on CPU?

9. Also, we signed up relatively late and didn't receive the $250 of AWS credits yet (we filled out the form). Could we get that resolved + the capacity reservation redeemable via credits?

10. While NKI 2 is adding features, can we use NKI 1 without penalty?

11. Briefly using NKIPy, it seems like there are many hidden nki.nisa functions we don't have documentation for. Is there any we can find those?

12. Are the instruction costs in NKI 2.26.1 accurate for Trainium 2 or NKI Beta 2?

13. I see a lot of max(MIN_II, N) as costs. Does this just mean free dimension must be at least MIN_II = 64 to make full use of an engine?
